tfx_addons/sampler/example/local_notebook.ipynb

```diff
@@ -0,0 +1,9670 @@
+{
```

Comment:
<p>after <code>context.run</code>  , let's use <code>context.show(statistics_gen.outputs['statistics'])</code>  to display the statistics</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tfx-addons/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>